qiita_db/support_files/patches/26.sql

```diff
@@ -0,0 +1,15 @@
+-- Jun 11, 2015
+
+-- Updating FASTA-Sanger -> FASTA_Sanger, needed so we can put restrictions on
+-- what kind of files the user can select in the GUI
+
+UPDATE qiita.filetype SET type='FASTA_Sanger' WHERE type = 'FASTA-Sanger';
```

Comment:
I don't fully understand why we need to change from `FASTA-Sanger` to `FASTA_Sanger`. Can you elaborate?